pkgs/tools/misc/atsi/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+, openssl
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "atsi";
+  version = "0.0.11";
+
+  src = fetchFromGitHub {
+    owner = "queer";
+    repo = pname;
+    rev = "0baf045c4de4862943e10d9a7bc28881f4706c71";
+    sha256 = "sha256-k0kD98MHN6AuX+W8VeiX6ExyaLTTZlE4/+5nQQF/x/g=";
```

Comment:
Prefer `hash = ...` to `sha256 = ...` Both currently work, but the former will make it easier if the algorithm ever needs to change.

Since there could be different hashing algorithms in use, this make sense.